### ISSUE TITLE
Geral

### ISSUE THREAD

**Issue by giovanaysandes** [giovanaysandes/Arduino_1#3]
#define pinPIR 7
#include <Servo.h>
Servo myservo;

const int servoPin = 8;
const int buttonPin = 11;
const int ledPin = 13;
int led =10;

void setup() {
pinMode(pinPIR, INPUT);
Serial.begin(9600);
pinMode(led, OUTPUT);
myservo.attach(servoPin);
pinMode(buttonPin, INPUT);
digitalWrite(buttonPin, HIGH);
pinMode(ledPin, OUTPUT);
digitalWrite(ledPin, LOW);
myservo.write(180);
delay(1000);
myservo.detach();
}

void loop() {

bool valorPIR = digitalRead(pinPIR);  
if (valorPIR) {
Serial.println("DETECTADO");

digitalWrite(led, HIGH);
delay(100); 
digitalWrite(led, LOW);
delay(100); 

} else {
Serial.println("--");
digitalWrite(led, LOW);
}

  int buttonVal = digitalRead(buttonPin);
  if(buttonVal == HIGH) {
    myservo.attach(servoPin);
    myservo.write(30);
    delay(575);
    myservo.write(180);
    delay(1500);
    myservo.detach();
    delay(5000); 
  }
  delay(13);

}
